src/components/DatasetDateItem/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import moment from 'moment';
+import { Tooltip, TooltipIcon } from '@cmsgov/design-system';
+
+type DateType = 'modified' | 'released' | 'refresh';
+
+type DatasetDateItemProps = {
+  /**
+   * Date label type
```

Comment:
If we need these comments could we abbreviate them to a single line

Reply:
removed comments

https://github.com/GetDKAN/cmsds-open-data-components/pull/293/commits/c786201aa591b9d4c5e851af28b8ce183e5678c6

Reply:
missed some of those comments. My apologies.

addressed
https://github.com/GetDKAN/cmsds-open-data-components/pull/293/commits/37ae1934c417deb857ef458f1a496e9c3484c4c1